.github/workflows/_build-cef.yml

```
@@ -0,0 +1,260 @@
+        run: |
+          $matches = Get-ChildItem -Recurse -Path "apps/desktop/target/release/bundle/nsis" -Filter "*-cef-setup.exe"
+          if ($matches.Count -eq 0) {
+            Write-Error "No *-cef-setup.exe artifact found!"
+            exit 1
+          }
+          Write-Host "Found CEF NSIS artifact: $($matches[0].FullName)"
+
+      - name: Sign Windows CEF executable
+        if: matrix.os == 'windows'
+        id: signpath-sign
+        uses: ./.github/actions/signpath-sign
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/signpath-sign": line 5: unexpected key "options" for definition of input "signpath-signing-policy"

(action)